pkg/console/subresource/configmap/configmap.go

```diff
@@ -49,15 +51,17 @@ func DefaultConfigMap(operatorConfig *operatorv1.Console, consoleConfig *configv
 	// Build a default config that can be overwritten from other sources
 	host := rt.Spec.Host
 	apiServerURL := getApiUrl(infrastructureConfig)
-	defaultConfig := NewYamlConfig(host, defaultLogoutURL, DEFAULT_BRAND, DEFAULT_DOC_URL, apiServerURL)
+	defaultConfig := NewYamlConfig(host, defaultLogoutURL, DEFAULT_BRAND, DEFAULT_DOC_URL, apiServerURL, defaultCustomProductName, defaultCustomLogoFile)
 	// Get console config from the openshift-config-managed namespace
 	extractedManagedConfig := extractYAML(managedConfig)
 
 	// Derive a user defined config using operator config
 	logoutRedirect := consoleConfig.Spec.Authentication.LogoutRedirect
 	brand := operatorConfig.Spec.Customization.Brand
 	docURL := operatorConfig.Spec.Customization.DocumentationBaseURL
-	userDefinedConfig := NewYamlConfig(host, logoutRedirect, brand, docURL, apiServerURL)
+	productName := operatorConfig.Spec.Customization.CustomProductName
```

Comment:
nit: I'd prob keep `custom` in the var name.  Thats the key thing here.
`customProductName`, or `customName` if you want to keep it short.